.vib/grafana/cypress/cypress/integration/grafana_test_suite.js

```diff
@@ -72,84 +71,83 @@ it('checks if smtp is configured', () => {
 
   cy.login();
   cy.visit('admin/settings');
-  cy.contains('td', SMTP_ADDRESS)
-    .should('exist');
-  cy.contains('td', SMTP_NAME)
-    .should('exist');
-  cy.contains('td', SMTP_USER)
-    .should('exist');
-})
+  cy.contains('td', SMTP_ADDRESS);
+  cy.contains('td', SMTP_NAME);
+  cy.contains('td', SMTP_USER);
+});
 
 it('checks if plugin page is showing plugins', () => {
   cy.login();
   cy.visit('/plugins?filterByType=datasource');
-  cy.get('[data-testid*="plugin-list"]')
-    .should('exist');
-})
+  cy.get('[data-testid*="plugin-list"]').should('exist');
+});
 
 it('checks if it is possible to create and delete a data source', () => {
   const DATASOURCE = 'Prometheus';
 
   cy.login();
   cy.visit('/datasources/new');
-  cy.get('input[placeholder*="Filter by name or type"]')
-    .type(DATASOURCE);
+  cy.get('input[placeholder*="Filter by name or type"]').type(DATASOURCE);
   cy.get(`button[aria-label*="Add data source ${DATASOURCE}"]`).click();
-  cy.contains('div', 'Datasource added')
-    .should('be.visible');
-    cy.visit("/datasources");
-    cy.contains('a', DATASOURCE).click({force:true});
-    cy.contains('button', 'Delete').click();
-    cy.contains('div', 'Are you sure you want to delete')
-      .should('be.visible');
-    cy.get('button[aria-label*="Confirm Modal Danger Button"]').should('be.visible').click();
-    verifySuccesOfAction();
-})
+  cy.contains('div', 'Datasource added');
+  cy.visit('/datasources');
+  cy.contains('a', DATASOURCE).click({ force: true });
+  cy.contains('button', 'Delete').click();
+  cy.contains('div', 'Are you sure you want to delete'); //Check how this works?
```

Comment:
Is this comment for future revisions or a typo of the current testing?

Reply:
It was a message from me, to me! Removed it!